dotnet-retire.sln

```diff
@@ -4,6 +4,10 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 VisualStudioVersion = 15.0.26430.6
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "SampleProjects", "SampleProjects", "{0FA6F455-D528-4C98-9635-271B4EB51ACC}"
+ProjectSection(SolutionItems) = preProject
+	SampleProjects\VulnerableSolution.sln = SampleProjects\VulnerableSolution.sln
+	SampleProjects\Vulnerable.Dockerfile = SampleProjects\Vulnerable.Dockerfile
```

Comment:
Is this one dead/useless now? Can we nuke the file?

Reply:
No, just leave it for now. I use the Docker files to test.